### PR TITLE
[INF-132] Add support for identity service

### PR DESCRIPTION
**Pull request by cheran-senthil**

### CODE REVIEW

**Comment by cheran-senthil** [comment]
Depends on https://github.com/AudiusProject/audius-protocol/pull/3303

File: audius-cli [hunk 1]
```diff
@@ -289,6 +292,29 @@ def restart(ctx, service, containers):
     )
 
 
+@cli.command()
+@click.argument("service", type=service_type, required=False)
+@click.argument("containers", type=container_type, nargs=-1)
+@click.pass_context
+def down(ctx, service, containers):
```
Comment (raymondjacobson) [inline audius-cli:299]: ty :)

File: identity-service/prod.env [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+solanaClaimableTokenProgramAddress=Ewkv3JahEFRKkcJmpoKB7pXbnUHwjAyXiwEo4ZY2rezQ
+solanaMintAddress=9LzCMqDgTKYz9Drzqnpgee3SGa89up3a247ypMj2xrqM
+solanaRewardsManagerProgramId=DDZDcYdQFEMwcu2Mwo75yGFjJ1mUQyyXLWzhZLEVFcei
+solanaRewardsManagerProgramPDA=71hWFVYokLaN1PNYzTAWi13EfJ7Xt9VbSWUKsXUT8mxE
+solanaRewardsManagerTokenPDA=3V9opXNpHmPPymKeq7CYD8wWMH8wzFXmqEkNdzfsZhYq
+userVerifierPublicKey=0xbeef8E42e8B5964fDD2b7ca8efA0d9aef38AA996
+web3Provider=https://poa-gateway.audius.co
+wormholeRPCHosts=https://wormhole-v2-mainnet-api.certus.one,https://wormhole.inotel.ro,https://wormhole-v2-mainnet-api.mcf.rocks,https://wormhole-v2-mainnet-api.chainlayer.network,https://wormhole-v2-mainnet-api.staking.fund,https://wormhole-v2-mainnet-api.chainlayer.network
+
+ethProviderUrl=
```
Comment (raymondjacobson) [inline identity-service/prod.env:34]: is the plan to leave these empty and make consumers required to fill this out in the override?

**Review by raymondjacobson — APPROVED** [review]

File: identity-service/prod.env (hunk 2 above)
Comment (cheran-senthil) [reply identity-service/prod.env:34]: Yes, that's right

**Pull request merged**